code/game/objects/items/weapons/grenades/chem_grenade.dm

```diff
@@ -240,20 +240,21 @@
 
 /obj/item/weapon/grenade/chem_grenade/adv_release // Intended for weaker, but longer lasting effects. Could have some interesting uses.
 	name = "advanced release grenade"
-	desc = "A custom made advanced release grenade. It is able to be triggered more than once. Can be configured using a multitool."
+	desc = "A custom made advanced release grenade. It is able to be detonated more than once. Can be configured using a multitool."
 	icon_state = "timeg"
 	var/unit_spread = 10 // Amount of units per repeat. Can be altered with a multitool.
 
 /obj/item/weapon/grenade/chem_grenade/adv_release/attackby(obj/item/I, mob/user, params)
 	if(istype(I, /obj/item/device/multitool))
-		if(unit_spread < 100)
+		if(unit_spread<25)
 			unit_spread += 5
+		if(unit_spread<100)

```

Comment:
Its a switch statement, not an if block, but i'll test real quick and make sure.